.env.local.example

```
@@ -15,7 +15,16 @@ NEXT_PUBLIC_LIVEKIT_URL=wss://livekit.3speak.tv
 # restarts but is XSS-readable for up to the JWT TTL (~24h).
 NEXT_PUBLIC_HANGOUTS_TOKEN_STORAGE=none
 
+# Hive Chat
+MONGODB_URI=mongodb://user:pass@host:27017/dbname?authSource=admin
+MONGODB_DB_NAME=snapiechat
+CHAT_JWT_SECRET=<random-secret-min-32-chars>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CHAT_JWT_SECRET key should go before the MONGODB_DB_NAME key

(UnorderedKey)